### PR TITLE
Jit examples

### PR DESCRIPTION
**Pull request by BenLloydPearson**
Screenshot 2023-06-06 at 10 58 47 AM
Screenshot 2023-06-06 at 10 58 41 AM
Screenshot 2023-06-06 at 10 58 29 AM
Screenshot 2023-06-06 at 10 58 22 AM

### CODE REVIEW

File: docs/automations/integrations/jit/review-jit-alerts/README.md
Comment (PavelLinearB) [inline docs/automations/integrations/jit/review-jit-alerts/README.md:15]: Shouldn't we want to require 2 reviewers in this case as well?

File: docs/downloads/automation-library/integrations/jit/review_jit_ignore_accept.cm
Comment (PavelLinearB) [inline docs/downloads/automation-library/integrations/jit/review_jit_ignore_accept.cm:9]: @BenLloydPearson this line (which i also published), has a bug:
The commenter that writes `#jit_ignore_accept` is not jit-ci, rather is the author of the PR, so in this case we should reject the comments made by jit-ci
in addition, map + match can be combined into a filter
so the line should be as follows:
      - {{ pr.conversations | reject(attr='commenter', term='jit-ci') | filter(attr='content', term='#jit_ignore_accept') | some }}

**Review by PavelLinearB — APPROVED** [review]
pre-approved the PR. will be available to merge after changes are fixed

File: docs/automations/integrations/jit/review-jit-alerts/README.md
Comment (BenLloydPearson) [reply docs/automations/integrations/jit/review-jit-alerts/README.md:15]: I mostly wanted to illustrate how to implement two separate policies based on the alert level, rather than to make definitive prescriptions about what the correct way to approach each situation is. In this situation, I also felt that requiring a review from the security team is a stronger enforcement than 2 reviews that could come from anyone, with the idea being that the PR author would be incentivized to fix the vulnerability rather than ping the security team.

However, If you feel strongly about the change, I'm happy to make it.

File: docs/downloads/automation-library/integrations/jit/review_jit_ignore_accept.cm
Comment (BenLloydPearson) [reply docs/downloads/automation-library/integrations/jit/review_jit_ignore_accept.cm]: Perfect, this could also explain some wonkiness I experienced while testing this. The most recent commit makes your requested change.

File: docs/automations/integrations/jit/review-jit-alerts/README.md
Comment (PavelLinearB) [reply docs/automations/integrations/jit/review-jit-alerts/README.md:15]: My only concern is that users will understand that it checks only for medium issues when requesting two assignees and only High for a reviewer.
Sometimes users may assume that medium issues include high as well

**Pull request merged**
